test/fixtures/import-module/expected.js

```diff
@@ -1,8 +1,10 @@
+"use strict";
+
 require("element-ui/lib/abc/style.css");
 
-var _Element = _interopRequireDefault(require("element-ui/lib/abc")).default;
+var _abc = _interopRequireDefault(require("element-ui/lib/abc"));
 
-function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }
 
-console.log(_Element.Button);
-console.log(_Element);
+console.log(_abc["default"].Button);
+console.log(_abc["default"]);
```

Comment:
This is currently rendered as `console.log(_Element);`. (while _Element is not defined)